.yamato/config/__shared.metafile

```diff
@@ -89,19 +89,20 @@ test_platforms:
       - [Win_DX12]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildWindowsDX12Linear"
       - [Win_Vulkan]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildWindowsVulkanLinear"
       
-      - [Win_DX11]: --extra-editor-arg="-playergraphicsapi=Direct3D11"
-      - [Win_DX12]: --extra-editor-arg="-playergraphicsapi=Direct3D12"
-      - [Win_Vulkan, Linux_Vulkan, Android_Vulkan]: --extra-editor-arg="-playergraphicsapi=Vulkan"
-      - [Android_OpenGLES3]: --extra-editor-arg="-playergraphicsapi=OpenGLES3"
-      - [iPhone_Metal]: --extra-editor-arg="-playergraphicsapi=Metal"
+      ### The method and values live in /com.unity.testframework.graphics/.../Editor/SetupProject.cs
+      - [Win_DX11]: --extra-editor-arg="-executemethod" --extra-editor-arg="SetupProject.ApplySettings" --extra-editor-arg="d3d11"
```

Comment:
I think this is fine but just in case I ask, are these intended with this PR (because they also come with the other PR)?

Reply:
ah yeah i did it in both because i thought i needed the changes in both - once i merge the other, this diff should disappear :)